src/main/java/com/deloitte/elrr/datasync/jpa/service/ImportService.java

```diff
@@ -43,4 +55,61 @@ public CrudRepository<Import, UUID> getRepository() {
     public UUID getId(final Import entity) {
         return entity.getId();
     }
+
+    /**
+     * @return importRecord
+     */
+    @Transactional(propagation = Propagation.REQUIRES_NEW)
+    public Import createImport() {
+        log.info("Creating new import.");
+        Import importRecord = new Import();
+        importRecord.setRecordStatus(StatusConstants.SUCCESS);
+        importRecord.setRetries(0);
+        importRecord.setImportName(StatusConstants.LRSNAME);
+        importRecord.setImportStartDate(initialDate);
+        importRecord.setImportEndDate(initialDate);
+        save(importRecord);
+        return importRecord;
+    }
+
+    /**
+     * @param importRecord
+     * @param status
+     * @return importRecord
+     * @throws ResourceNotFoundException
+     */
+    @Transactional(propagation = Propagation.REQUIRES_NEW)
+    public Import updateImportStatus(Import importRecord, String status)
+            throws ResourceNotFoundException {
+
+        log.info("Updating import status to " + status);
+
+        importRecord.setRecordStatus(status);
+        update(importRecord);
+
+        return importRecord;
+    }
+
+    /**
+     * @param importRecord
+     * @return importRecord
+     * @throws ResourceNotFoundException
+     */
+    @Transactional(propagation = Propagation.REQUIRES_NEW)
```

Comment:
why would you need a new transaction scope here? We should chat about this and what it does and why it is very dangerous.